apps/src/code-studio/components/Libraries/LibraryClientApi.js

```diff
@@ -0,0 +1,64 @@
+import clientApi from '@cdo/apps/code-studio/initApp/clientApi';
+
+const LIBRARY_NAME = 'library.json';
+export default class LibraryClientApi {
+  constructor(channelId) {
+    this.channelId = channelId;
+    this.libraryApi = clientApi.create('/v3/libraries');
+  }
+
+  publish(library) {
+    this.libraryApi.put(
+      this.channelId,
+      JSON.stringify(library),
+      LIBRARY_NAME,
+      (error, data) => {
+        if (error) {
+          // In the future, errors will be surfaced to the user in the publish dialog
+          console.warn('Error publishing library: ' + error);
+        }
+      }
+    );
+  }
+
+  getLatest() {
+    this.libraryApi.fetch(
+      this.channelId + '/' + LIBRARY_NAME,
+      (error, data) => {
+        if (data) {
+          // in the future, responses will be passed back to the import dialog
+          console.log('Library: ' + data);
+        } else {
+          // In the future, errors will be surfaced to the user in the import dialog
+          console.warn('Error getting library: ' + error);
+        }
+      }
+    );
+  }
+
+  getVersion(versionId) {
+    let library;
+    this.libraryApi.fetch(
+      this.channelId + '/' + LIBRARY_NAME + '?version=' + versionId,
+      (error, data) => {
+        if (data) {
+          // in the future, responses will be passed back to the import dialog
```

Comment:
Are these ToDos tracked?

Reply:
Yes - this is just setting up some basic scaffolding. I expect this to evolve a bit as the libraries feature goes through a couple churns. I'll expect to finish these up as part of these two tasks:
https://codedotorg.atlassian.net/browse/STAR-710
https://codedotorg.atlassian.net/browse/STAR-709